server/src/repositories/version-history.repository.ts

```diff
@@ -20,7 +20,7 @@ export class VersionHistoryRepository implements IVersionHistoryRepository {
     return this.db.selectFrom('version_history').selectAll().orderBy('createdAt', 'desc').executeTakeFirst();
   }
 
-  @GenerateSql({ params: [DummyValue.STRING] })
+  @GenerateSql({ params: [{ version: DummyValue.STRING }] })
```

Comment:
Noticed this was broken when generating SQL 😅

Reply:
Lol that explains why it was missing. Why doesn't it error on that?

Reply:
Lol that explains why it was missing. Why doesn't it error on that?

Reply:
some queries throw an error. like when you try to insert a fake uuid into the album_assets intersection table. when you run `make sql` it logs these errors, and there was one for this query.